test/correctly/import_nested/proto/greet/v1/common/common.proto

```
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package common;
```

Buf (1.55.1)

Files with package "common" must be within a directory "common" relative to root but were in directory "test/correctly/import_nested/proto/greet/v1/common".

(PACKAGE_DIRECTORY_MATCH)